src/components/Overview/ImageCarousel.jsx

```diff
@@ -71,19 +73,23 @@ const ImageCarousel = ({currentStyle, setExpanded, expanded, zoomed, setZoomed})
       })}
 
       <Arrow
+        aria-label='downArrow'
         className={zoomed ? 'ZoomArrow' : 'downArrow'}
         style={{visibility: imageInd < miniUrlArr.length - 1 ? 'visible' : 'hidden' }}
         onClick={() => {setImage(imageInd + 1)}}
         ><FaAngleDown /></Arrow>
     </div>
 
       <Arrow
+        aria-label='LeftArrow'
         className={zoomed ? 'ZoomArrow' : expanded ? 'leftArrowExp' : 'leftArrow'}
         style={{visibility: imageInd !== 0 ? 'visible' : 'hidden' }}
         onClick={() => {setImage(imageInd - 1)}}
       ><FaAngleLeft /></Arrow>
 
       <Img
+        data-testid='testImage'
+        alt='product image'
```

Comment:
So, the alt text kind of needs to describe what's in the image - the idea is that it's for screen readers so if someone who has bad eyesight is using one, they can still know what's on the page in front of them - I would think your alt text could just be dynamically render the product name, though